crucible/src/Lang/Crucible/Simulator/ExecutionTree.hs

```diff
@@ -1075,7 +1076,29 @@ ctxSymInterface = to (\ctx ->
 functionBindings :: Lens' (SimContext p sym ext) (FunctionBindings p sym ext)
 functionBindings = lens _functionBindings (\s v -> s { _functionBindings = v })
 
--- | Access the custom user-state inside the 'SimContext'.
+-- | Custom state inside the 'SimContext'.
+--
+-- Crucible itself is entirely polymorphic over @p@. Downstream applications can
+-- instantiate it to any sort of state that they would like to associate with a
+-- 'SimContext'.
+--
+-- For example, applications based on @macaw-symbolic@ can instantiate this to a
```

Comment:
Please include a hyperlink for `macaw-symbolic`.

Reply:
Whoops, thanks for the catch!